_episodes/04-recipe.md

```diff
@@ -29,7 +29,7 @@ recipes that are shipped with ESMValTool, type
 esmvaltool recipes list
 ```
 
-We will start by running [examples/recipe_python.yml](https://docs.esmvaltool.org/en/latest/recipes/recipe_python.html)
+We will start by running [examples/recipe_python.yml](https://github.com/ESMValGroup/ESMValTool/blob/master/esmvaltool/recipes/examples/recipe_python.yml)
```

Comment:
I'd prefer to refer to the documentation rather than the source code. Would this be okay as well?

```suggestion
We will start by running [examples/recipe_python.yml](https://docs.esmvaltool.org/en/latest/recipes/recipe_examples.html)
```

Reply:
Ideally the documentation changes less frequently than the code, so this should work. However I think we also wanted to give a pointer to where they could see the recipe and other recipes and also get  a sense of where the recipe is placed in the directory structure better. If that's not so much a requirement then we can have it point to the documentation.